src/test/java/com/google/devtools/build/lib/syntax/util/EvaluationTestCase.java

```diff
@@ -219,6 +219,14 @@ public void checkEvalErrorContains(String msg, String... input) throws Exception
     }
   }
 
+  public void checkEvalErrorDoesNotContain(String msg, String... input) throws Exception {
+    try {
+      eval(input);
```

Comment:
If you expect an exception, maybe call fail() after eval()?

Reply:
I don't necessarily expect an exception; perhaps renaming this method would help.  Want I want to assert is that either the eval succeeds, or if it fails, it fails with a different error.

The specific use here is that load statements in this test always fail, because the files being loaded don't exist.  Rather than hard-coding that error message, I'm using this helper to assert that the failure isn't because of the incompatible label.  If the test suite were fixed so load()s worked, these tests wouldn't have to be changed.

I'm open to a different approach, or a different name that makes this more clear.